projects/Agorakit/default.nix

```
@@ -0,0 +1,14 @@
+{
+  pkgs,
+  lib,
+  sources,
+} @ args: {
+  packages = {
+    inherit (pkgs) agorakit;
+  };
+  nixos = {
+    modules.services.agorakit = "${sources.inputs.nixpkgs}/nixos/modules/services/web-apps/agorakit.nix";
+    tests.agorakit = null;
```

```suggestion
    tests = null;
```

Our invocation of the tests fails because you gave it `tests.agorakit` that was supposed to be a test but is `null`. Our filter only removes `tests` if the whole attribute set is `null`.

---

It doesn't seem to be that though. Maybe a Nixpkgs bump is needed then?